tests/ui/static/use_of_mutable_static_unsafe_fn.e2015.stderr

```diff
@@ -0,0 +1,20 @@
+error: use of mutable static is discouraged
+  --> $DIR/use_of_mutable_static_unsafe_fn.rs:14:14
+   |
+LL |     let _y = &X;
+   |              ^^ use of mutable static
+   |
+   = note: use of mutable static is a hard error from 2024 edition
+   = note: mutable statics can be written to by multiple threads: aliasing violations or data races will cause undefined behavior
+note: the lint level is defined here
+  --> $DIR/use_of_mutable_static_unsafe_fn.rs:8:9
+   |
+LL | #![deny(static_mut_ref)]
+   |         ^^^^^^^^^^^^^^
+help: use `addr_of_mut!` macro
+   |
+LL |     let _y = addr_of_mut!(X);
```

Comment:
Seems strange to suggest `addr_of_mut!` when the user wrote a *shared* reference